slither/solc_parsing/declarations/function.py

```diff
@@ -690,12 +794,26 @@ def _parse_catch(self, statement: Dict, node: NodeSolc) -> NodeSolc:
         else:
             params = statement[self.get_children("children")]
 
-        if params:
-            for param in params.get("parameters", []):
-                assert param[self.get_key()] == "VariableDeclaration"
-                self._add_param(param)
+        params = params["parameters"] if params and "parameters" in params else None
+
+        if (not is_try_clause) and params:
+            assert (len(params) == 1)
+            parameter = params[0]
+            assert parameter[self.get_key()] == "VariableDeclaration"
+
+            initialValue = self._get_default_initial_value(statement["src"], parameter["typeName"]["name"])
```

Comment:
I think the catch-clause parameters should be treated just like normal function parameters. They don't need to be initiailzed and must be considered to have unknown values in all detectors. Assigning contrived value would cause other issues down the road. We may introduce a fake library function such `function unknownInt()` in our CFG. But, I'd like to avoid that, if possible.

Let's find out how function parameters are handled by the detector and make sure that catch-parameters are handled the same way.

Reply:
Since the Slither CFG doesn't include any notion of scoped variables, I think we will need to create the fake library function `unknownInt()`.

Another option would be to extend the CFG to include a notion of scoped variables, but this would probably be too drastic of a change.